docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -51,79 +51,60 @@ def store_files(files):
 
 ### Defining basic dependencies for graph-backed assets
 
-The `from_graph` attribute on the `AssetsDefinition` object infers upstream and downstream asset dependencies from the graph definition provided. In the most simple case when the graph returns a singular output, Dagster infers the name of the graph to be the outputted asset key.
+Similar to with basic software-defined assets, Dagster infers the upstream assets from the names of the arguments to the decorated function.
 
-In the example below, Dagster creates an asset with key `middle_asset` from the `middle_asset` graph. Just like assets defined via <PyObject object="asset" decorator />, each argument to the decorated graph function is an upstream asset name. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
+The example below includes an asset named `middle_asset`. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
 
 ```python file=/concepts/assets/graph_backed_asset.py startafter=start_basic_dependencies endbefore=end_basic_dependencies
-from dagster import AssetsDefinition, asset, graph
+from dagster import asset, graph_asset, op
 
 
 @asset
 def upstream_asset():
     return 1
 
 
-@graph
-def middle_asset(upstream_asset):
-    return add_one(upstream_asset)
+@op
+def add_one(input_num):
+    return input_num + 1
 
 
-middle_asset = AssetsDefinition.from_graph(middle_asset)
+@op
+def multiply_by_two(input_num):
+    return input_num * 2
+
+
+@graph_asset
+def middle_asset(upstream_asset):
+    return multiply_by_two(add_one(upstream_asset))
 
 
 @asset
 def downstream_asset(middle_asset):
-    return middle_asset + 1
+    return middle_asset + 7
 ```
 
-When your graph returns multiple outputs, Dagster infers each output name to be the outputted asset key. In the below example, `two_assets_graph` accepts `upstream_asset` and outputs two assets, `first_asset` and `second_asset`:
+### Graph-backed multi-assets
+
+Using the <PyObject object="graph_multi_asset" decorator />, you can create a combined definition of multiple assets that are computed using the same graph of ops and same upstream assets.
+
+In the below example, `two_assets` accepts `upstream_asset` and outputs two assets, `first_asset` and `second_asset`:
 
 ```python file=/concepts/assets/graph_backed_asset.py startafter=start_basic_dependencies_2 endbefore=end_basic_dependencies_2
-from dagster import AssetsDefinition, GraphOut, graph
+from dagster import AssetOut, graph_multi_asset
 
 
-@graph(out={"first_asset": GraphOut(), "second_asset": GraphOut()})
-def two_assets_graph(upstream_asset):
+@graph_multi_asset(outs={"first_asset": AssetOut(), "second_asset": AssetOut()})
+def two_assets(upstream_asset):
     one, two = two_outputs(upstream_asset)
     return {"first_asset": one, "second_asset": two}
-
-
-two_assets = AssetsDefinition.from_graph(two_assets_graph)
-```
-
-### Defining explicit dependencies for graph-backed assets
```

Comment:
I think it would be helpful to still include a brief example of how you can use the same `@graph` to create multiple `AssetsDefinition`s via `AssetsDefinition.from_graph`. This is a fairly common pattern and I think it'll help with understandability to at least briefly include how `keys_by_output_name` can be used

Reply:
That is fair - I will try to take this on in a followup